src/foodbox.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+function Foodbox(props) {
+  const [quantity, setQuantity] = useState(props.food.quantity);
+
+  const handleQuantity = (event) => {
+    event.preventDefault();
+    setQuantity(event.target.value);
+  };
+
+  let foodForTotal = {
+    name: props.food.name,
+    quantity,
+    calories: quantity * props.food.calories,
+  };
+
+  const addToTotal = (event) => {
+    event.preventDefault();
```

Comment:
Same comment as above.

Reply:
Thanks for the feedback Lloyd. The classes were a bit carelessly copied from Bulma I have to admit.. 
All is resolved now!